source/opt/block_merge_util.cpp

```diff
@@ -171,8 +171,15 @@ void MergeWithSuccessor(IRContext* context, Function* func,
       // flow declaration.
       context->KillInst(merge_inst);
     } else {
+      // Move OpLine/OpNoLine information to merge_inst. This solves
+      // the validation error that OpLine is placed between OpLoopMerge
+      // and OpBranchConditional.
+      auto terminator = bi->terminator();
+      merge_inst->dbg_line_insts() = terminator->dbg_line_insts();
```

Comment:
This is going to replace the merge instruction debug line instructions, should it be appending instead?

Reply:
Good point. Updated.